build.sh

```diff
@@ -1,49 +1,62 @@
-#!/bin/bash
+#! /usr/bin/env bash
```

Comment:
nit: `#!/usr..` without space?

The interpreter starts right after the `#!`, I'm not sure if this makes a practical difference

Reply:
oh yes the space is a bad habit from some very old age that belong to the earlier history of unix (so it precedes me), it can be dropped even if supported, it makes no sense to add it, it's just my fingers that are adding them in an automated manner :p